sdk/python/kfp/v2/compiler_cli_tests/compiler_cli_tests.py

```diff
@@ -36,12 +36,16 @@ def _test_compile_py_to_json(self, file_base_name):
         golden = json.load(f)
         # Correct the sdkVersion
         golden['sdkVersion'] = 'kfp-{}'.format(kfp.__version__)
+        # Need to sort the list items before comparison
+        golden['tasks'].sort(key=lambda x: x['executorLabel'])
```

Comment:
I think `assertDictEqual` does not require consistent key order. Does it fail without this statement?

Reply:
Ah I see. Thanks!